docs/zone-pivot-groups.yml

```diff
@@ -0,0 +1,13 @@
+# YamlMime:ZonePivotGroups
```

Comment:
I can't see this working on the article on the review site

Reply:
The review site has been very evil with .NET today.. I have a ticket filed. Things keep working and not working. The images sometimes render, sometimes not. It's weird. I just get a spinning load icon for a while and then all the timeouts trigger. For example, I took this screenshot earlier:

![image](https://user-images.githubusercontent.com/2672110/67890224-8c63ad00-fb0d-11e9-9581-bf6b7dc02c1c.png)

But now it's not rendering. you can look at the dev tools and see if things are failing. I don't even see the TOC at all.

![image](https://user-images.githubusercontent.com/2672110/67893090-2bd76e80-fb13-11e9-8908-66e7db7c6ee8.png)

Reply:
Exactly. For me it just showed all content in one page with no tabs. 😊